java/Oleksii-Riabov/src/com/zoolatech/lecture10/tasks/_1/Task1.java

```diff
@@ -0,0 +1,106 @@
+package com.zoolatech.lecture10.tasks._1;
```

Comment:
please create separate feature branches for your home tasks; as for now, you use a same branch for all works

Reply:
Okey. Should I remove "lecture 10" task from current pull request?

Reply:
no, let's keep it as is